### PR TITLE
Html5 video element lazyloading

### PR DESCRIPTION
**Pull request by Levdbas**
Hi,

As mentioned in https://github.com/ApoorvSaxena/lozad.js/issues/127 lazyloading of videos would be very nice to have in this library. I created a small addition to the code to load all video source elements. by including the `lozad` class on a `video` element and give all `source` elements the `data-src` attribute the video will now lazyload all sources available. Autoplay works as well. The demo has been updated as well with both a normal video and a autoplay video. Video used in the demo is available under Creative Commons Attribution 3.0.

If any additional work is needed, let me know. Since this is my first PR, let me know if something is missing.

### CODE REVIEW

**Comment by ApoorvSaxena** [comment]
@Levdbas thanks for your contribution, it will be great to have new test cases accompany the functionality that you've added

File: src/lozad.js [hunk 1]
```diff
@@ -19,6 +19,15 @@ const defaultConfig = {
       }
       element.appendChild(img)
     }
+    if (element.nodeName.toLowerCase() === 'video' && !element.getAttribute('data-src')) {
+      if (element.children) {
+        const childs = element.children
+        for (let i = 0; i <= childs.length - 1; i++) {
+          childs[i].src = childs[i].getAttribute('data-src')
+        }
+        element.load()
```
[the remark this answers is not in the document]
Comment (Levdbas) [reply src/lozad.js]: the load() function is needed to actually trigger the loading of the video, see https://stackoverflow.com/questions/34033115/delay-loading-of-html5-video as well. Just changing the data-src to src isn't enough.

File: demo/index.html [hunk 2]
```diff
@@ -191,6 +191,36 @@ <h1>&lt;iframe> example</h1>
...
+    <div class="sub-main">
+      <article class="main-heading">
+        <h1>&lt;video> example</h1>
+        <p>video courtesy of <a href="https://www.bigbuckbunny.org/" target="_blank">Big Buck Bunny</a>.</p>
+      </article>
+    </div>
+    <div id="video">
+      <article class="frame">
+          <div class="video-example video-example-1">
+            <video width="560" height="315" controls autoplay loop muted class="lozad-picture">
```
[the remark this answers is not in the document]
Comment (Levdbas) [reply demo/index.html]: I agree, but it is nice to see that autoplay actually works as well with lazyloaded videos. I'll let you decide if it belongs in the demos.

[the remark this answers is not in the document]
Comment (ApoorvSaxena) [reply demo/index.html]: let's ignore it for now

**Pull request merged**